pisek/jobs/parts/task_job.py

```diff
@@ -183,3 +190,25 @@ def _diff_files(self, file_a: TaskPath, file_b: TaskPath) -> str:
             stdout=subprocess.PIPE,
         )
         return diff.stdout.decode("utf-8")
+
+    def _file_content(
```

Comment:
This method name sounds like it returns the entire file. Wouldn't something like `_quote_file` be better?